### ISSUE TITLE
records named "exports"

### ISSUE THREAD

**Issue by chipee** [ngoduykhanh/PowerDNS-Admin#468]
Hello everyone,

pdns version : 4.0.3
pdns Admin build (I'm not sure): 787bdba9e147

Having a domain with a records named: "exports" create a situation where the records listing page cannot be render properly. There is a JavaScript error in the browser console:
```
ReferenceError: module is not defined[Learn More] main.js:3091:36
TypeError: $(...).DataTable is not a function
```
So let say for example, I have a domain test.test. If i create a record exports.test.test when I'll go to the record page of this domain the page crash (eg. No buttons works and the paging functionality do not show up).
With error:
![error](https://user-images.githubusercontent.com/23584873/53274798-969ff280-36c6-11e9-82b2-b20cfc1911d0.jpeg)

Correct one:
![correct](https://user-images.githubusercontent.com/23584873/53274777-7f610500-36c6-11e9-816a-db12703d99d2.jpeg)

It took me a while to figure that out since it is my first deployment of PowerDNS-Admin and the domain in question has about a thousand of entries. I hope it can help someone else in the same situation.

Thanks for your awesome work!

**Comment by ngoduykhanh** [comment]
It is because the edit/delete button id has the value `exports` that might used by another component. Fix in #472

**Comment by chipee** [comment]
It work perfectly.
Big thanks for your work.
